src/app/services/plugins.service.ts

```diff
@@ -37,7 +33,7 @@ export interface PluginDescription {
     description: string;
     tags: string[];
 
-    running: PluginStatus;
+    running?: PluginStatus;
```

Comment:
The comment before was plural to indicate all three added parameters (also `timeAgo` and `olderThan24`) as all of them are information that is filled in later.